drop-boxes/test-dropbox/etl_msconvert.py

```diff
@@ -1,487 +0,0 @@
-"""
-This etl script converts proteomics raw files to mzML.
-
-Incoming raw files are converted and the new mzML is copied
```

Comment:
why did you remove the file?

Reply:
it's not used and the dropbox in question is for testing anyway